Sources/NostrSDK/Events/ReactionEvent.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  File.swift
+//  
+//
+//  Created by Terry Yiu on 8/12/23.
+//
+
+import Foundation
+
+/// A reaction event (kind 7) in response to a different event.
+///
+/// See [NIP-25](https://github.com/nostr-protocol/nips/blob/master/25.md).
+public class ReactionEvent: NostrEvent {
+    public var reactedEventId: String? {
+        tags.last(where: { $0.name == .event })?.value
```

Comment:
Why `last` on these? Is that intentional (as opposed to `first`)?

Reply:
It's intentional per the spec.

Reply:
https://github.com/nostr-protocol/nips/blob/master/25.md#tags

```
The last e tag MUST be the id of the note that is being reacted to.
The last p tag MUST be the pubkey of the event being reacted to.
```